src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/utils/StringHelper.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.utils
+
+import org.springframework.web.server.ServerWebInputException
+import java.util.UUID
+
+object StringHelper {
+  @JvmStatic
```

Comment:
the idiomatic way to do this in kotlin is using a companion object

i.e. 
```
class StringHelper {
  companion object {
    fun parseId() {
    ...
```

this way there's no need for the `@JvmStatic` annotation either

Reply:
nitpick: since this is no longer a private method in a single controller, a more descriptive method name or small docstring comment might be useful.

Reply:
Thanks - I didn't know the kotlin way